chipy_org/apps/job_board/models.py

```diff
@@ -130,3 +122,47 @@ def days_elapsed(self):
             return days_elapsed_from_posting
         else:
             return None
+
+    @property
+    def expiration_date(self):
+
+        # expiration_date shows up as a field in the admin panel.
+        # If post is approved and the approval_date is set,
+        # compute the expiration_date.
+        if self.status == "AP" and self.approval_date:
+            expiration_date = self.approval_date + datetime.timedelta(days=self.days_to_expire)
+            return expiration_date
+        else:
+            return None
+
+    @classmethod
+    def approved_posts(cls):
+
+        # I've split these into two queries in anticipating that there might be
+        # different ordering or filtering based on sponsored vs non-sponsored
```

Comment:
maybe this should read approved_and_active to denote that they are in an approved state, but they also have not expired.